webservices/resources/reports.py

```diff
@@ -73,6 +73,12 @@ def get_range_filters():
     ]
     return filter_range_fields
 
+def get_match_filters():
+    filter_match = [
+        ('filer_type', models.CommitteeReports.means_filed)
+    ]
+    return filter_match
```

Comment:
Shot in the dark for the issue I see below, but what if this were called `filter_match_fields`?  It shouldn't matter, but it is a convention used across all other filter definitions.